src/components/avatar/avatar.ts

```diff
@@ -15,11 +18,16 @@ export class Avatar extends Block {
   }
   protected render(): string {
     const { userName, srcAvatar } = this.state;
+
     // language=hbs
     return `
       <img
         class="avatar"
-        src="${srcAvatar}"
+        src="${
+          srcAvatar !== 'undefined' && srcAvatar !== 'null'
```

Comment:
Думаю что можно упростить это не потеряв логику
```suggestion
          !srcAvatar
```